lib/Screens/dashboard.dart

```diff
@@ -1,34 +1,49 @@
 import 'package:flutter/material.dart';
 import 'package:pispapp/Screens/account_dashboard.dart';
-import 'package:pispapp/Screens/new_payment.dart';
+import 'package:pispapp/Screens/splash.dart';
+import 'package:pispapp/widgets/bottom_navigation.dart';
+
+class Dashboard extends StatefulWidget {
+  @override
+  _DashboardState createState() => _DashboardState();
+}
+
+class _DashboardState extends State<Dashboard> {
+  Widget widgetToShow = AccountDashboard();
+  void changeRootWidget(Widget widget) {
+    setState(() {
+      widgetToShow = widget;
+    });
+  }
+
+  static List<BottomNavigationButton> buttons = <BottomNavigationButton>[
+    BottomNavigationButton(
+      Icons.arrow_forward,
+      SplashScreen(),
+    ),
```

Comment:
I feel like this menu shouldn't be here ... Isn't the splash screen only for users who just downloaded the app and haven't signed in?